APE0.rst

```diff
@@ -0,0 +1,385 @@
+
+Term and Active/Emeritus Status
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Voting Members have no term or term limits. Voting Members who have stopped
+contributing are encouraged to declare themselves as "emeritus.” Those who have
+not made any significant contribution for two years may be asked to move
+themselves to the “emeritus” category by the Coordination Committee. If no
+response is received, the Coordination Committee may automatically change a
+Voting Member’s status to emeritus. To record and honor their contributions,
+emeritus Voting Members will continue to be listed. emeritus Voting Members are
+not able to participate in votes.
+
+Removing Voting Members
+^^^^^^^^^^^^^^^^^^^^^^^
+In exceptional circumstances, it may be necessary to remove someone from the
+Voting Members against their will. A vote must be held to remove a Voting
+Member. Such a vote is triggered by a motion made by an active Voting Member,
+which must be seconded by an additional Voting Member. The vote must conclude
+no more than three months after the motion is seconded. Removal requires approval
+by two-thirds of all active Voting Members at the time the motion is made. The
+motion, second, and vote will be held in private. Removal under this provision
+will be reflected by updating the list of Voting Members.
+
+It may be necessary for the Ombudsperson and the Coordination Committee to
+remove a Voting Member for violations of the Code of Conduct. In this case, the
+Coordination Committee and Ombudsperson will work together to make this
+decision.
+
+Approving and Modifying This Charter
+------------------------------------
+This document was submitted following the process in `APE 1`_, and the normal APE
+acceptance procedures will be followed. The Coordination Committee at the time
+of submitting this APE are all co-authors and therefore will not override any
+consensus of the community on accepting the final version.
+
+Changes to this Charter after it has been accepted should follow the
+modification process in `APE 1`_, with the exception that the final approval of the
+modification requires approval by a two-thirds vote of the Voting Members rather
+than approval by the Coordination Committee.
+
+Attribution and Acknowledgments
+-------------------------------
+The format and some of the structures outlined in this document are heavily
+inspired by the Python Language Governance structure
+(`PEP 13 <https://www.python.org/dev/peps/pep-0013/>`_), the YT Project's Team
+Infrastructure
+(`YTEP 1776 <https://ytep.readthedocs.io/en/latest/YTEPs/YTEP-1776.html>`_), and
+earlier less-formal descriptions of the Astropy governance.
+
+.. _APE 1: https://github.com/astropy/astropy-APEs/blob/master/APE1.rst
+
+
+Branches and Pull Requests
+==========================
+N/A
+
+Implementation
+==============
+This Charter enters into force upon this APE being accepted (see the last
+section of the description). At that time the ``GOVERNANCE.md`` file in the
```

Comment:
This file is .rst, not .md - should this be `GOVERNANCE.rst`?

Reply:
```suggestion
section of the description). At that time the ``GOVERNANCE.rst`` file in the
```
Good catch!